scanapp/views.py

```diff
@@ -178,6 +145,50 @@ def post(self, request):
         )
 
 
+class UrlScanView(FormView):
+    template_name = 'scanapp/urlscan.html'
+    form_class = UrlScanForm
+
+    def post(self, request, *args, **kwargs):
+        form = self.form_class(request.POST)
+
+        if form.is_valid():
+            url = request.POST['url']
+            logger = logging.getLogger(__name__)
+
+            if str(request.user) == 'AnonymousUser':
```

Comment:
Do not use str() ... not here and almost nowhere. Use the attribute (e.g. here likely reques.user.name ?)